src/EnergyPlus/OutputReportTabular.hh

```diff
@@ -489,6 +491,86 @@ namespace OutputReportTabular {
         }
     };
 
+    struct compLoadsSurface
+    {
+        Real64 loadConvectedNormal = 0.0;
+        Real64 loadConvectedWithPulse = 0.0;
+        Real64 netSurfRadSeq = 0.0;
+        Real64 ITABSFseq = 0.0;     // used for determining the radiant fraction on each surface
+        Real64 TMULTseq = 0.0;      // used for determining the radiant fraction on each surface
+        Real64 lightSWRadSeq = 0.0; // short wave visible radiation
+        Real64 feneSolarRadSeq = 0.0;
+    };
+
+    struct compLoadsTimeStepSurfaces
+    {
+        std::vector<compLoadsSurface> surf;
+    };
+
+    struct componentLoadsSurf
+    {
+        std::vector<compLoadsTimeStepSurfaces> ts;
+    };
```

Comment:
New sets of nested vectors to replace a bunch of loose `Array3D`. This is for surfaces, similar ones for zone/spaces, and enclosures.